apps/src/code-studio/progressRedux.js

```diff
@@ -136,6 +145,11 @@ export const updateFocusArea = (changeFocusAreaPath, focusAreaPositions) => ({
 
 export const showTeacherInfo = () => ({ type: SHOW_TEACHER_INFO });
 
+/**
+ * Authorizes the user to be able to see lockable stages
+ */
+export const authorizeLockable = () => ({ type: AUTHORIZE_LOCKABLE });
```

Comment:
This may eventually go in a different reducer just for tracking lockable related info, but this seems like an okay place for it to start.